scripts/network/database/database.gd

```diff
@@ -26,13 +26,41 @@ func init() -> bool:
 
 
 func create_account(username: String, password: String) -> bool:
-	if username == "" or password == "":
+	if username == "" or password == "" or not is_account_valid(username, password):
 		J.logger.info("Invalid username or password")
 		return false
 
 	return backend.CreateAccount(username, password)
 
 
+func is_account_valid(username: String, password: String) -> bool:
```

Comment:
I think we will want a way at some point to bubble up the exact error messages to inform the user as to why their account creation failed. Can do here, or pop a new issue and address later. 

A possible solution could be:

```
func is_account_valid(username: String, password: String) -> Dictionary:
  ...
  if username.length() < 4 or username.length() > 16:
    var validation_error_message := "Username must be at least 4 characters long or maximum 16 characters long."
    J.logger.warn(validation_error_message)
    return {"validation_status": false, "validation_error_message": validation_error_message}
  ...
```

Or there may be some other solution. I wish GDscript supported multi-value returns, but from what I see it does not. It'd be nice to do

```
func foo() -> (ret_val, message)

var return_val, message = foo()
```

But I don't think we have that, so for multi-var returns its either an array or dictionary.